site/demos/airbeds/webpack.config.js

```diff
@@ -1,7 +1,6 @@
 const path = require('path');
 
 module.exports = {
-	mode: 'development',
```

Comment:
Instead of changing mode for each demo and example, we can fix webpack version in site, which actually throws error regarding mode key
https://github.com/appbaseio/reactivesearch/blob/next/site/package.json#L35

Just make sure site build and start are not failing

Reply:
Done the changes https://github.com/appbaseio/reactivesearch/pull/936